api/app/sfms/raster_addresser.py

```diff
@@ -81,6 +99,16 @@ def get_calculated_precip_key(self, datetime_to_calculate_utc: datetime):
         calculated_weather_prefix = f"{self.weather_model_prefix}/{datetime_to_calculate_utc.date().isoformat()}/"
         return os.path.join(calculated_weather_prefix, compose_computed_precip_rdps_key(datetime_to_calculate_utc))
 
+    def get_daily_ffmc(self, timestamp: datetime, run_type: RunTypeEnum):
```

Comment:
I was using `get_uploaded_index_key` for this part, maybe makes sense to split them out though, whatever you think works best

Reply:
`get_uploaded_index_key` is based on an enum that contains only the abbreviated FWI names, whereas SFMS gives us the full FWI name for FFMC, so I opted for a specific function.